### PR TITLE
Ensure ansible output is parseable

### PR DESCRIPTION
**Pull request by ssbarnea**
Closes: https://github.com/ansible/vscode-ansible/issues/373
Closes: #173
Closes: #194

### CODE REVIEW

**Review by ganeshrn — APPROVED** [review]
Please do add changelog fragement

**Review by priyamsahoo — APPROVED** [review]
LGTM !

File: docs/changelog-fragments.d/236.bugfix.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Ensure that Ansible calls do not return ANSI escapes, so we can parse them.
+[vscode-ansible#373] -- by {user}`ssbarnea`
```
Comment (webknjaz) [inline docs/changelog-fragments.d/236.bugfix.md:2]: This is the wrong way of linking issues. The fragment should've just been symlinked.

File: docs/changelog-fragments.d/236.bugfix.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Ensure that Ansible calls do not return ANSI escapes, so we can parse them.
```
Comment (webknjaz) [inline docs/changelog-fragments.d/236.bugfix.md:1]: Don't forget to review for the use of the past tense references. Changelog shouldn't look like a TODO-list.

**Pull request merged**